chainstory/linkapp/consumers.py

```diff
@@ -0,0 +1,17 @@
+from channels.generic.websocket import AsyncWebsocketConsumer
+import json
+
+class MyConsumer(AsyncWebsocketConsumer):
+    async def connect(self):
+        await self.accept()
+
+    async def disconnect(self, close_code):
+        pass
+
+    async def receive(self, text_data):
+        text_data_json = json.loads(text_data)
+        message = text_data_json['message']
+
+        await self.send(text_data=json.dumps({
```

Comment:
What does self.send do?

Reply:
I do believe that's a built method to AsyncWebsocketConsumer but I haven't figured out how it works quite yet